public/docs/_examples/pipes/dart/web/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Pipes Example</title>
+    <link rel="stylesheet" href="style.css">
+    <script async src="main.dart" type="application/dart"></script>
+    <script async src="packages/browser/dart.js"></script>
+</head>
+<body>
+<hero-birthday></hero-birthday>
+<chained-pipes></chained-pipes>
+<power-booster></power-booster>
+<power-boost-calculator></power-boost-calculator>
+<my-hero></my-hero>
+<heroes-list></heroes-list>
+</body>
+</html>
```

Comment:
Make sure all files end w/ a newline. It makes git happy.

Notice the little :no_entry_sign: ?